pyproject.toml

```diff
@@ -4,3 +4,4 @@ known_first_party = "mentat"
 
 [tool.ruff]
 line-length = 120
+ignore = ["E731"]
```

Comment:
Added this rule to ignore lambda assignment (https://beta.ruff.rs/docs/rules/lambda-assignment/) as we have these everywhere and imo they're ok as long as you don't have a ton of logic in them.